source/bricklaunch

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+#
+# bricklaunch.sh <ops> <program>
+# - open a new vt and switch to it
+# - give 'user' ownership of the vt
+# - run the given programm on the new vt as 'user'
+#
+
+OPTIND=1
+user='user'
+openvt_opts=""
+rm_ttyfile=0
+
+function show_help {
+  cat <<END_OF_HELP
+bricklaunch.sh <opts> <program>
+
+  -h        this help
+  -s        switch to the new virtual terminal
```

Comment:
We're doing something like that with brickman already.

https://github.com/ev3dev/brickman/blob/ev3dev-jessie/debian/brickman.service#L10

We could reserve tty2 for OpenRoberta.